src/middlewares/task.middleware.js

```diff
@@ -93,7 +95,10 @@ export default (store: any) => (next: any) => (action: any) => {
           });
 
           child.on('exit', code => {
-            const wasSuccessful = code === 0 || code === null;
+            // For Windows Support
+            // Windows sends code 1 (I guess its because we foce kill??)
+            const successfullCode = isWin ? 1 : 0;
+            const wasSuccessful = code === successfullCode || code === null;
```

Comment:
Hm, I think the issue here is that we lose the ability to detect true failures from false ones; if the task crashes, it would still display "Task Completed" for Windows.

That said, I don't see an easy solution to the problem, it doesn't look like `taskkill` supports specifying the exit code, and the current architecture means there's no clear way to communicate.

I don't think this is a showstopper; if we can get everything else working on Windows, we can deal with fixing this issue in a subsequent PR (likely by rearchitecting it so that `ABORT_TASK` handles this directly, instead of using the child-process listeners as an event bus)

Reply:
I think following up state of the process together with a custom redux variable might be beneficial. We can check if the termination was triggered with the switch or the process ended unexpectedly.

I don't know how much of this can be implemented but there can be lots of different informative states. For example rather than just having Running/Idle/Failed we can maybe have Started/Running/Stopping/Stopped/Idle/Failed/Ended Unexpectedly. Since Guppy targets new users, being more informative IMO a great approach.